cmd/gctcli/commands.go

```diff
@@ -4084,3 +4083,146 @@ func getHistoricCandles(c *cli.Context) error {
 	jsonOutput(result)
 	return nil
 }
+
+var getHistoricCandlesExtendedCommand = cli.Command{
+	Name:      "gethistoriccandlesextended",
+	Usage:     "gets historical candles extended for the specified granularity up to range size time from now.",
```

Comment:
Pedantic nit, but for all of our other usage commands we don't have a period at the end